yt/frontends/chimera/tests/test_outputs.py

```diff
@@ -0,0 +1,195 @@
+        ("chimera", "v_csound"),
+        ("chimera", "wBVMD"),
+        ("chimera", "w_c"),
+        ("chimera", "ye_c"),
+        ("chimera", "ylep"),
+        ("chimera", "z_nuc_rep_c"),
+    ]
+    assert_equal(str(ds), "chimera_00001_grid_1_01.h5")
+    assert_equal(ds.geometry, "spherical")  # Geometry
+    assert_almost_equal(
+        ds.domain_right_edge,
+        ds.arr([1.03651796e10 - 34, 3.14159265e00, 6.28318531e00], "code_length"),
+    )  # domain edge
+    assert_array_equal(
+        ds.domain_left_edge, ds.arr([0.0, 0.0, 0.0], "code_length")
+    )  # domain edge
+    assert_array_equal(ds.domain_dimensions, np.array([722, 240, 1]))  # Dimensions
+    assert_array_equal(ds.field_list, _fields)
+
+    dd = ds.all_data()
+    for field in _fields:
+
+        def field_func(name):
+            return dd[field]
+
+        if field != ("chimera", "shock"):
+            yield GenericArrayTest(ds, field_func, args=[field])
+
+
+Three_D = "C15-3D-3deg/chimera_002715000_grid_1_01.h5"
+
+
+@requires_ds(Three_D)
+def test_3D():
+    ds = data_dir_load(Three_D)
+    _fields = [
+        ("chimera", "a_nuc_rep_c"),
+        ("chimera", "abar"),
+        ("chimera", "ar36"),
+        ("chimera", "be_nuc_rep_c"),
+        ("chimera", "c12"),
+        ("chimera", "ca40"),
+        ("chimera", "cr48"),
+        ("chimera", "dudt_nu"),
+        ("chimera", "dudt_nuc"),
+        ("chimera", "e_book"),
+        ("chimera", "e_int"),
+        ("chimera", "e_rms_1"),
+        ("chimera", "e_rms_2"),
+        ("chimera", "e_rms_3"),
+        ("chimera", "e_rms_4"),
+        ("chimera", "entropy"),
+        ("chimera", "fe52"),
+        ("chimera", "grav_x_c"),
+        ("chimera", "grav_y_c"),
+        ("chimera", "grav_z_c"),
+        ("chimera", "he4"),
+        ("chimera", "lumin_1"),
+        ("chimera", "lumin_2"),
+        ("chimera", "lumin_3"),
+        ("chimera", "lumin_4"),
+        ("chimera", "mg24"),
+        ("chimera", "n"),
+        ("chimera", "ne20"),
+        ("chimera", "ni56"),
+        ("chimera", "nse_c"),
+        ("chimera", "num_lumin_1"),
+        ("chimera", "num_lumin_2"),
+        ("chimera", "num_lumin_3"),
+        ("chimera", "num_lumin_4"),
+        ("chimera", "o16"),
+        ("chimera", "p"),
+        ("chimera", "press"),
+        ("chimera", "rho_c"),
+        ("chimera", "s32"),
+        ("chimera", "si28"),
+        ("chimera", "t_c"),
+        ("chimera", "ti44"),
+        ("chimera", "u_c"),
+        ("chimera", "v_c"),
+        ("chimera", "v_csound"),
+        ("chimera", "wBVMD"),
+        ("chimera", "w_c"),
+        ("chimera", "ye_c"),
+        ("chimera", "ylep"),
+        ("chimera", "z_nuc_rep_c"),
+        ("chimera", "zn60"),
+    ]
+    assert_equal(str(ds), "chimera_002715000_grid_1_01.h5")
+    assert_equal(ds.geometry, "spherical")  # Geometry
+    print(
+        ds.domain_right_edge
+        - ds.arr([1.06500257e09, 3.14159265e00, 6.28318531e00], "code_length")
+    )
```

Comment:
failed to catch this earlier but please make sure there are no print statements in your tests